ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -938,11 +937,10 @@ data TraceIteratorEvent blk
 -- This is the usual case and innocent while caught-up; but while syncing, it
 -- means that we are downloading blocks at a smaller rate than we can validate
 -- them, even though we generally expect to be CPU-bound.
-data TraceChainSelStarvationEvent blk
-    -- | A ChainSel starvation started at the given time.
-  = ChainSelStarvationStarted Time
-
-    -- | The last ChainSel starvation ended at the given time as a block wth the
-    -- given point has been received.
-  | ChainSelStarvationEnded Time (RealPoint blk)
+--
+-- Somehow it happens regularly during syncing for very short times.
```

Comment:
> Somehow it happens regularly during syncing for very short times.

This sentence seems a little out of place. Should it be a TODO (investigate why we're seeing these hiccups)? An Issue?